packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -178,6 +181,7 @@ export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
               <div className="flex flex-1 flex-col">
                 <SearchPanelAction provider={SearchProviderEnum.Posts} />
                 <SearchPanelAction provider={SearchProviderEnum.Chat} />
+                <SearchPanelTagSuggestions title="Tags" />
```

Comment:
I would suggest to conditionally render this based on the feature flag, rather then having the flag within the component

Reply:
This would then enrol users any time they see search panel (eg. any time they open our app) vs how it is handled now it only enrols users when they actually search and interact with a panel.

Reply:
I'll actually use the `useConditionalFeature` hook, that will make it better.